conda-build/common/delete_old_night_packages.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
+import re
+import argparse
+
+MAX_NUMBER_OF_PACKAGES = 30
+
+parser = argparse.ArgumentParser("Running benchmarks on simulator")
```

Comment:
Description is copied?
```suggestion
parser = argparse.ArgumentParser("Deleting old Conda packages.")
```